packages/ui-components/src/checkins.ts

```diff
@@ -0,0 +1,174 @@
+import hyper from "@macrostrat/hyper";
+import styles from "./checkins.module.sass";
+import { LngLatCoords } from "@macrostrat/map-interface";
+import { useDarkMode } from "@macrostrat/ui-components";
+import { Icon } from "@blueprintjs/core";
+import mapboxgl from "mapbox-gl";
+
+const h = hyper.styled(styles);
+
+const rockdApiURL = "https://dev.rockd.org/api/v2";
+
+function BlankImage({ src, className, width, height, onClick, onError, alt }) {
+    console.log(src)
+    return h("img", {src: src, className, width, height, onClick, onError, alt})
+}
+
+function getImageUrl(person_id, photo_id) {
+    return rockdApiURL + "/protected/image/" + person_id + "/thumb_large/" + photo_id;
+}
+
+function getProfilePicUrl(person_id) {
+    return rockdApiURL + "/protected/gravatar/" + person_id;
+}
+
```

Comment:
We generally don't want to hard-code this type of config (e.g., what URLs to hit for images; API addresses) in the web-components repository